hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieCreateHandle.java

```diff
@@ -62,7 +65,7 @@ public class HoodieCreateHandle<T, I, K, O> extends HoodieWriteHandle<T, I, K, O
   protected long recordsDeleted = 0;
   private Map<String, HoodieRecord<T>> recordMap;
   private boolean useWriterSchema = false;
-  private final boolean preserveMetadata;
+  private boolean preserveMetadata;
```

Comment:
why reverting this ?

Reply:
Removed this change. I had addded it for fixing one of the tests. Not required anymore.